### PR TITLE
Deployment strategy changes

### PR DESCRIPTION
**Pull request by igreulich**
## Description
Added some prod flags to keep the world from seeing our form before we are ready to show it to them.

## Testing done
Local prod and dev builds, UI compared.

## Screenshots
n/a

## Acceptance criteria
- [x] Configure 4142 for deployment

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/526EZ/config/form.js [hunk 1]
```diff
@@ -686,25 +683,83 @@ const formConfig = {
...
-              }.view:selectableEvidenceTypes.view:privateMedicalRecords`,
-              formData,
-            );
-            const requestsRecords =
-              _.get(
-                `disabilities.${index}.view:uploadPrivateRecords`,
-                formData,
-              ) === 'no';
-            return hasRecords && requestsRecords;
+            const isNotProd = __BUILDTYPE__ !== 'production';
```
Comment (aaronjlech) [inline src/applications/disability-benefits/526EZ/config/form.js]: You're checking the Prodtype in 3 locations, and setting a variable here only. Would it be beneficial to set this variable in the scope of the file and then reuse within config as needed?

**Comment by igreulich** [comment]
@aaronjlech Fixed, take a look again, please.

**Comment by aaronjlech** [comment]
@igreulich LGTM

**Review by aaronjlech — APPROVED** [review]

**Review by imeecuison — APPROVED** [review]
Looks good!

**Comment by igreulich** [comment]
I don't really know what is wrong with the Jenkins build. The errors SEEM to be emitting from the design system, possibly with the way some props are getting passed in to some react components.

There is no real indication of where the failures are occurring, only they they are surfacing in React, and moment.

**Pull request closed without merging**